tests/storage/export.go

```diff
@@ -562,6 +562,7 @@ var _ = SIGDescribe("Export", func() {
 			fileAndPathName,
 		}
 		By(fmt.Sprintf("Downloading from URL: %s", downloadUrl))
+		Eventually(ThisPod(downloadPod), 30*time.Second, 1*time.Second).Should(HaveConditionTrue(k8sv1.PodReady))
```

Comment:
Wouldn't it be cleaner to let k8s take care of this, by passing `curl...` as the `command` of the container of the pod?

Reply:
It's nicer, but it could hide bugs since it's more forgiving. Some tests also rely on stdout in this suite which is going to be nastier to assert over if we stop using the exec API.

Anyway, I am seeing that some tests in other suites are failing with
```
error dialing backend: write tcp: use of closed network connection
```
Right after a ready assertion, so this may invalidate this PR